### DISCUSSION TITLE
（求助）帮帮我，绳网先生

### DISCUSSION THREAD

**Discussion by xingjianyadegou** [share121/inter-knot#922] [其他]
![1723287411894](https://github.com/user-attachments/assets/8c8e9667-1c86-4a3f-8544-71abb181bf68)
朋友们，我是抽朱鸢还是青衣

**Comment by MiDang123** [comment]
抽新不抽旧，听我的，青衣冲就完事了（错了当我没说，欸嘿

**Comment by juyab** [comment]
喜欢什么抽什么
